package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "robotics-route-planner",
+  "version": "1.0.0",
+  "description": "Route planner is a tool that helps with planning robot routes for a variety of robotics challenges.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node server.js",
+    "dev": "node server.js",
+    "test": "echo \"No tests specified\" && exit 0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Eee0001/robotics-route-planner.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Eee0001/robotics-route-planner/issues"
+  },
+  "homepage": "https://github.com/Eee0001/robotics-route-planner#readme"
+}
```

Comment:
I'd recommend running `npm install express` in your main repo directory, which will install the `express` library as a dependency for this library. When I ran this locally, it gave the `Error: Cannot find module 'express'` error and refused to run. You probably have express installed globally, which is also fine!

To make sure anyone can use this app, you can identify express as a dependency by running:

`npm install express`

Then in the future, if anyone runs `npm install` in your project directory, npm will make sure that the library is installed and available. Super small change, but maybe tackle that as a follow-up PR.